.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+*.swp
```

Comment:
```suggestion
```

`*.swp` are files which your editor makes. It's not specific to this project.

Please use your global gitignore (`$XDG_CONFIG_HOME/git/ignore`, see https://git-scm.com/docs/gitignore) instead. I would like to avoid receiving lots of pull requests from many users of various editors / IDEs.